mailbox/plugin/deleted-messages-vault/src/main/java/org/apache/james/vault/blob/BlobStoreVaultGarbageCollectionTask.java

```diff
@@ -90,4 +90,12 @@ public TaskType type() {
     public Optional<TaskExecutionDetails.AdditionalInformation> details() {
         return Optional.of(new AdditionalInformation(beginningOfRetentionPeriod, deletedBuckets));
     }
+
+    ZonedDateTime getBeginningOfRetentionPeriod() {
+        return beginningOfRetentionPeriod;
+    }
+
+    Flux<BucketName> getRetentionOperation() {
```

Comment:
you can't expose it that way. Flux is a stateful thing. It's already dangerous to take the Flux as an input parameter but it's even more dangerous to return it to the outside.

Reply:
removed

Reply:
it looks to be still here